cdk/helpers/lambdas/checkSumOfFiles.spec.ts

```diff
@@ -1,9 +1,11 @@
 import path from 'node:path'
 import { checkSumOfFiles } from './checksumOfFiles.js'
+import assert from 'node:assert'
+import { describe, test as it } from 'node:test'
 
-describe('checkSumOfFiles()', () => {
-	it('should calculate a checksum of files', async () =>
-		expect(
+void describe('checkSumOfFiles()', () => {
```

Comment:
```suggestion
void describe('checkSumOfFiles()', () => 
```

The bracket could be removed because the scope of that function is just another function. Same with other similar cases. Check https://github.com/NordicSemiconductor/asset-tracker-cloud-coiote-aws-converter-js/pull/85/commits/6041492da7ab06e8a41143e91dabcd6284c64f17 if you would like more info